capsules/src/ros.rs

```diff
@@ -0,0 +1,142 @@
+//! Read Only Systems calls
+//!
+//! This capsule provides read only system calls to userspace applications.
+//! This is similar to the Linux vDSO syscalls.
+//!
+//! The benefit of using these is that applications can avoid the context
+//! switch overhead of traditional syscalls by just reading the value from
+//! memory.
+//!
+//! The value will only be as accurate as the last time the application was
+//! switched to by the kernel.
+//!
+//! The layout of the read only syscalls in the allow region depends on the
+//! version. Userspace can use `command 0` to get the version information.
+//!
+//! Versions are backwards compatible, that is new versions will only add
+//! fields, not remove existing ones or change the order.
+//!
+//! Version 1:
+//!   |-------------------------|
+//!   |       Count (u32)       |
```

Comment:
Count of what?

Reply:
It's the count of context switchs. I have updated this